dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/WCServerTest.java

```diff
@@ -407,19 +356,43 @@ public void test_ServletMapping() throws Exception {
         logmsg = wlp.waitForStringInLogUsingMark("SRVE9016E:");
         Assert.assertNotNull("TestServletMapping application deployment did not result in  message SRVE9016E: ", logmsg);
 
+        // Need to sleep because it times out waiting for the config update in logs while server is still doing cleanup/spinup stuff
+        Thread.sleep(5000);
```

Comment:
Would we be able to avoid this sleep if we `waitForConfigUpdateInLogUsingMark` was used for the first config update?